core/mutator.js

```diff
@@ -204,7 +209,7 @@ Blockly.Mutator.prototype.resizeBubble_ = function() {
     this.workspaceHeight_ = height;
     // Resize the bubble.
     this.bubble_.setBubbleSize(width + doubleBorderWidth,
```

Comment:
Add first param to the next line.

Reply:
I did, but I also accidentally duplicated it.  Whoops.  Fixed.